src/frontend/next/src/pages/index.tsx

```diff
@@ -8,6 +9,7 @@ const Home = () => {
         <title>Create Next App</title>
         <link rel="icon" href="/favicon.ico" />
       </Head>
+      <Header />
```

Comment:
I think @PedroFonsecaDEV is also working on this, so you might want to avoid and have him do it in #1531.  @PedroFonsecaDEV, agree?

Reply:
@humphd @jossiey 
Agree.

Reply:
I am a bit unclear on what needs to be done here. @humphd @PedroFonsecaDEV can anybody clarify this for me - what should be done as #1548 got merged?

Reply:
@izhuravlev 
After porting the `Header` to TS, you will need to import and use it in the `pages/_app.tsx` instead of `pages/index.tsx`

I'm available to make a call if you need more context and clarifications. Please let me know on our slack channel!